From db8cb882fb743a216687abc5cf372302405c1053 Mon Sep 17 00:00:00 2001
From: Igor Gnatenko <i.gnatenko.brain@gmail.com>
Date: Sat, 7 Dec 2019 12:39:53 +0100
Subject: [PATCH] Remove usage of jemallocator

It is PITA to package it in Fedora, so we will have to have fd a bit
slower for now.

Signed-off-by: Igor Gnatenko <i.gnatenko.brain@gmail.com>
---
 src/main.rs | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/src/main.rs b/src/main.rs
index 426eb08..c4af1ef 100644
--- a/src/main.rs
+++ b/src/main.rs
@@ -35,11 +35,6 @@ use crate::internal::{
     pattern_has_uppercase_char, transform_args_with_exec, FileTypes,
 };
 
-// We use jemalloc for performance reasons, see https://github.com/sharkdp/fd/pull/481
-#[cfg(all(not(windows), not(target_env = "musl")))]
-#[global_allocator]
-static ALLOC: jemallocator::Jemalloc = jemallocator::Jemalloc;
-
 fn main() {
     let checked_args = transform_args_with_exec(env::args_os());
     let matches = app::build_app().get_matches_from(checked_args);
-- 
2.24.0

